site/gatsby-site/src/components/submissions/SubmissionReview.js

```diff
@@ -83,6 +87,29 @@ const SubmissionReview = ({ submission }) => {
       },
     });
 
+  const subscribeToNewReports = async (incident_id) => {
```

Comment:
Is there a rule that checks that only subscriptions of the current user are allowed?

Reply:
Yes, on `customData/subscriptions/rules.json` we have a rule for that:
```
{
    "name": "Owner of the document",
    "apply_when": {
        "userId": "%%user.id"
    },
    "read": true,
    "write": true,
    "insert": true,
    "delete": true,
    "search": true
}
```